submission of A0 and A1

cchheerilyn opened the pull request:
my solutions for A0 and A1

cchheerilyn commented:
@mtw or @s-will, please review

s-will commented on A0/cchheerilyn-HelloWorld.sh:
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/bash
+
+echo "Hello, world!"
```
Be precise. According to my specification, this should print "Hello World!"

s-will commented on A0/cchheerilyn-HelloWorld.sh:
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/bash
+
+echo "Hello, world!"
+cat $1
```
```suggestion
cat "$1"
```

s-will commented on A1/cchheerilyn-WordCount.py:
```diff
@@ -0,0 +1,36 @@
+
+wordcount = 0
+
+file = input("Please enter a filename: ")
```
One must parse the command line arguments to get the filename; have a look into Python's argparse module

s-will commented on A1/cchheerilyn-WordCount.py:
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+print ("A1 - Word Count")
```
This should not be printed according to my specification; we will always specify exactly what the programs should read and write

s-will commented on A1/cchheerilyn-WordCount.py:
```diff
@@ -0,0 +1,36 @@
+for line in f:#.lower():
+    text = line.split()
+    wordcount += len(text)
+print("total word count: \t", wordcount)
```
again, check the specification in which format this should be printed

s-will commented on A1/cchheerilyn-WordCount.py:
```diff
@@ -0,0 +1,36 @@
+        else:
+            uniques [word] = 1
+    return uniques
+print ("unique words: ", uniquewordcount (text))
```
same here

cchheerilyn commented:
Dear s-will,

Im sorry I wasnt able to attend todays lecture due to a home emergency. I dont know what the consequences will be but either way I think i‘m a little overwhelmed by the workload/requirements in programming skills as u might have seen in my solution for A1. So I guess it‘s better for me to quit your course.

Kind regards, 
Cherilyn

> On 14.03.2019, at 08:48, Sebastian Will <notifications@github.com> wrote:
> 
> @s-will requested changes on this pull request.
> 
> I commented on some obvious things. Naturally, other things (mostly concerning coding style) could be improved as well; we will discuss many such things in class anyway.
> 
> In A1/cchheerilyn-WordCount.py:
> 
> > @@ -0,0 +1,36 @@
> +#!/usr/bin/env python
> +
> +print ("A1 - Word Count")
> +
> +#For programming warm up, here is the "Word Count" challenge:
> +#Write a program that reads in a text (file name given on the command line) and counts the total number of words and the number of different words. On request, the program should print a list of the words.
> +
> +wordcount = 0
> +
> +file = input("Please enter a filename: ")
> One must parse the command line arguments to get the filename; have a look into Python's argparse module
> 
> In A1/cchheerilyn-WordCount.py:
> 
> > @@ -0,0 +1,36 @@
> +#!/usr/bin/env python
> +
> +print ("A1 - Word Count")
> This should not be printed according to my specification; we will always specify exactly what the programs should read and write
> 
> In A1/cchheerilyn-WordCount.py:
> 
> > +wordcount = 0
> +
> +file = input("Please enter a filename: ")
> +f = open (file, 'r')
> +
> +#convert all letters to lower case, replace special characters with space
> +for line in f:
> +    small = f.lower()
> +    for char in '"\',.:;?!%^()[]{}|-_+=$<>\n':
> +        text = small.replace (char, ' ')
> +
> +#counts and prints total word count
> +for line in f:#.lower():
> +    text = line.split()
> +    wordcount += len(text)
> +print("total word count: \t", wordcount)
> again, check the specification in which format this should be printed
> 
> In A1/cchheerilyn-WordCount.py:
> 
> > +#counts and prints total word count
> +for line in f:#.lower():
> +    text = line.split()
> +    wordcount += len(text)
> +print("total word count: \t", wordcount)
> +
> +#counts and prints unique word count
> +def uniquewordcount (input):
> +    uniques = {}
> +    for word in input:
> +        if word in uniques:
> +            uniques [word] += 1
> +        else:
> +            uniques [word] = 1
> +    return uniques
> +print ("unique words: ", uniquewordcount (text))
> same here
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
